docs/4.0/utilities/image-replacement.md

```diff
@@ -18,3 +18,18 @@ Utilize the `.text-hide` class or mixin to help replace an element's text conten
   @include text-hide;
 }
 {% endhighlight %}
+
+The `.text-hide` class is useful for when you want the benefits heading tags like accessibility and SEO, but want to utilize your brand's logo image instead of text.
```

Comment:
Small edit for this:

```
Use the `.text-hide` class to maintain the accessibility and SEO benefits of heading tags, but want to utilize a `background-image` instead of text.
```